### PR TITLE
Add reference xml; edit units 

### PR DESCRIPTION
**Pull request by StefaniaGrimaldi**

### CODE REVIEW

File: 2_ESSENTIAL_setting-file/index.md [hunk 1]
```diff
@@ -670,7 +670,7 @@ Note that a couple of variables can be [initialized internally](https://ec-jrc.g
 	</textvar>                                                          
 	<textvar name="ThetaForestInit1Value" value="-9999">            
 	<comment>                                                           
-	initial soil moisture content layer 1                                 
+	initial soil moisture content layer 1                                 https://ec-jrc.github.io/lisflood-code/3_step3_preparing-setting-file/
```
Comment (domeniconappo) [inline 2_ESSENTIAL_setting-file/index.md:673]: @StefaniaGrimaldi 
Maybe you added this link by mistake?

**Review by domeniconappo — COMMENTED** [review]
Thanks for this!
Could you just review and answer my comment?

File: 2_ESSENTIAL_setting-file/index.md (hunk 1 above)
Comment (StefaniaGrimaldi) [reply 2_ESSENTIAL_setting-file/index.md:673]: Yes, that is a mistake.

Comment (domeniconappo) [reply 2_ESSENTIAL_setting-file/index.md:673]: yes, just try to do another commit

**Review by domeniconappo — APPROVED** [review]

**Pull request merged**